packages/schemix/src/schemix/columns.py

```
@@ -0,0 +1,444 @@
+from __future__ import annotations
+
+import datetime
+from abc import ABC, abstractmethod
+from typing import Any, Self, TypeGuard
```

Ruff (0.11.9)

`typing.TypeGuard` imported but unused

Remove unused import: `typing.TypeGuard`

(F401)